### PR TITLE
Adds a feature request for pausing currently playing music on screen lock

### PR DESCRIPTION
**Pull request by BarakaAka1Only**
This is a feature which can be viewed here: 

https://github.com/radiant-player/radiant-player-mac/issues/607

image

Go to the Preferences -> General -> Check on Toggle current music play status

You can test it in REAL time without having to reload the Radiant Player App; do control + shift + eject or press Login Window :

image

Both will trigger the music to PAUSE and PLAY if there is any item that is playing of-course

Screen Locked  = PAUSE music

Screen Unlocked  = PLAY music

and if you uncheck  Toggle current music play status you will see it off added for users that don't want to use this option Default state is off

### CODE REVIEW

File: radiant-player-mac/Preferences/PreferencesWindowController.xib [hunk 1]
```diff
@@ -256,8 +237,37 @@
                         <binding destination="Qr1-oe-yDR" name="value" keyPath="automaticallyDownloadsUpdates" id="9Ul-Aq-ZHq"/>
                     </connections>
                 </button>
+                <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" setsMaxLayoutWidthAtFirstLayout="YES" id="g2c-F2-be6">
+                    <rect key="frame" x="146" y="17" width="314" height="39"/>
+                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                    <textFieldCell key="cell" controlSize="small" sendsActionOnEndEditing="YES" allowsEditingTextAttributes="YES" id="tAg-Di-ow3">
+                        <font key="font" metaFont="smallSystem"/>
+                        <string key="title">Applying this will allow you to toggle the current music playing to be paused on a locked screen and plays back on a unlocked screen</string>
```
Comment (tjanson) [inline radiant-player-mac/Preferences/PreferencesWindowController.xib]: May I suggest a slightly less verbose explanation text? Something like:

> Pause playback when screen is locked and resume when screen is unlocked.

Comment (BarakaAka1Only) [inline radiant-player-mac/Preferences/PreferencesWindowController.xib]: Sure i can adjust that, thank you!

Comment (BarakaAka1Only) [inline radiant-player-mac/Preferences/PreferencesWindowController.xib]: @tjanson added the adjustment :)

**Comment by jacobwgillespie** [comment]
👍

**Pull request merged**